css-view-transitions-2/Overview.bs

```diff
@@ -875,6 +875,9 @@ It has the following [=struct/items=]:
 
 	: <dfn>initial snapshot containing block size</dfn>
 	:: a [=tuple=] of two numbers (width and height).
+
+	: <dfn>resolved transition</dfn>
```

Comment:
This doesn't look right. This data structure is meant to be serialized across Documents and I don't think we can or want to serialize the script "ViewTransition" object?

Reply:
It's null when you serialize it from the old document... Also serialization is somewhat of an implementation concern.
We can make this something that's "associated" with the view transition params but I don't think it matters too much.